dags.yaml

```diff
@@ -1359,3 +1359,19 @@ bqetl_crash:
     start_date: "2023-12-10"
   tags:
     - impact/tier_2
+
+bqetl_use_counter_analysis:
+  schedule_interval: 0 8 * * *
+  default_args:
+    depends_on_past: false
+    email:
+      - kwindau@mozilla.com
```

Comment:
should also include the telemetry mailing group, here's an example:
https://github.com/mozilla/bigquery-etl/blob/main/dags.yaml#L1320

Reply:
thanks! fixed this!